android/src/main/res/layout/card_view_channel.xml

```diff
@@ -1,69 +1,67 @@
 <?xml version="1.0" encoding="utf-8"?>
-<LinearLayout
-    xmlns:android="http://schemas.android.com/apk/res/android"
+<LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
     xmlns:tools="http://schemas.android.com/tools"
+    android:id="@+id/channel_container"
     android:layout_width="match_parent"
     android:layout_height="match_parent"
-    android:id="@+id/channel_container"
-    android:orientation="vertical">
+    android:clickable="true"
+    android:foreground="?attr/selectableItemBackground"
+    android:orientation="vertical"
+    android:stateListAnimator="@anim/animation">
```

Comment:
can u please clarify what does this do?

Reply:
android:stateListAnimator="@anim/animation" is not needed as well i was trying added animation in a different way